.changeset/plenty-pants-fold.md

```diff
@@ -0,0 +1,67 @@
+---
+"@neo4j/graphql": patch
+---
+
+Add `unsafeEscapeOptions` to `Neo4jGraphQL` features with the following flags:
+
+-   `disableRelationshipTypeEscaping` (default to `false`)
+-   `disableLabelEscaping` (defaults to `false`)
+
+These flags remove the automatic escaping of node labels and relationship types in the generated Cypher.
+
+For example, given the following schema:
+
+```graphql
+type Actor {
+    name: String!
+}
+
+type Movie {
+    title: String!
+    actors: [${Actor}!]! @relationship(type: "FROM_PRODUCTION]->(:Production)-[:ACTED_IN", direction: OUT)
+}
+```
+
+A GraphQL query going through the `actors` relationship:
+
+```graphql
+query {
+    movies {
+        title
+        actors {
+            name
+        }
+    }
+}
+```
+
+Will normally generate the following Cypher for the relationship:
+
+```cypher
+MATCH (this:Movie)-[this0:`FROM_PRODUCTION]->(:Production)-[:ACTED_IN`]->(this1:Actor)
```

Comment:
I'm unsure what's the best way to document this changeset.
What I'm worried about this phrasing is that it makes it unclear whether this hack is officially supported or not.
As a personal choice, I would use vague language similar to the language used in the respective Cypher Builder PR https://github.com/neo4j/cypher-builder/pull/505 .
Curious to hear @darrellwarde  opinion, and happy with whatever conclusion.